visualization/app/codeCharta/codeCharta.component.html

```diff
@@ -5,17 +5,6 @@
 	<ribbon-bar-component class="ribbon-bar-component"></ribbon-bar-component>
 </div>
 
-<div id="actionButtons">
```

Comment:
remove unused css code for actionButtons as well

Reply:
guys, you just deleted our `unfocus` button..

Reply:
I added the actionButtons code again in #639